src/controllers/channel.controller.js

```diff
@@ -0,0 +1,31 @@
+const Channel = require('../models/channel.models');
+
+const {
+    response_200,
+    response_400,
+    response_500
+} = require('../utils/responseCodes.utils');
+
+exports.createChannel = async (req,res) => {
+    try {
+        const {type, description, name} = req.body;
+        if(!type || !description || !name || !req.params)
```

Comment:
You are getting the mongo object of workspace and user. They are verified earlier by some middleware. So do not use req.params.